### PR TITLE
docs: fix description of SelectInitialValue story

### PR DESCRIPTION
**Pull request by layershifter**
## New Behavior

Description matches implementation.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `60.071 kB`<br />`16.745 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `187.768 kB`<br />`52.723 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `34.427 kB`<br />`11.354 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.069 kB`<br />`2.053 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |   `23.963 kB`<br />`8.206 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/SelectField.fixture.js'>SelectField</abbr>                                                                          |  `32.961 kB`<br />`10.616 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/8ca9702605078cfdd584d75b5ea469066b2a4349'>8ca9702605078cfdd584d75b5ea469066b2a4349</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/Avatar-mount_base.html">1617</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/Avatar-mount.html">1611</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/Button-mount_base.html">1172</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/Button-mount.html">1171</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/FluentProvider-mount_base.html">1950</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/FluentProvider-mount.html">1986</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/FluentProviderWithTheme-mount_base.html">784</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/FluentProviderWithTheme-mount.html">769</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">725</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">720</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">775</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">766</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/InfoButton-mount_base.html">702</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/InfoButton-mount.html">707</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/MakeStyles-mount_base.html">2377</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/MakeStyles-mount.html">2356</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/Persona-mount_base.html">3711</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/Persona-mount.html">3632</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/SpinButton-mount_base.html">3056</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26493/perf-test-react-components/SpinButton-mount.html">3155</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by layershifter** [comment]
@microsoft/cxe-coastal friendly ping.

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Comment by layershifter** [comment]
@microsoft/cxe-red @khmakoto @behowell can you please check the PR?

File: packages/react-components/react-select/stories/Select/SelectInitialValue.stories.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ export const InitialValue = () => {
 InitialValue.parameters = {
   docs: {
     description: {
-      story: 'A Select can have its initial value defined by using `selected` on a child `option`',
+      story: 'A Select can have its initial value defined by using `defaultValue` prop.',
```
Comment (behowell) [inline packages/react-components/react-select/stories/Select/SelectInitialValue.stories.tsx:22]: nit wording:
```suggestion
      story: 'A Select can have its initial value defined by using the `defaultValue` prop.',
```

**Review by behowell — APPROVED** [review]

**Review by khmakoto — APPROVED** [review]

**Pull request merged**